examples/shell/cmd_base64.cpp

```diff
@@ -15,13 +15,13 @@
  *    limitations under the License.
  */
 
-#include <shell/shell.h>
+#include <lib/shell/shell.h>
```

Comment:
this `lib/` element should be in the include directives

Reply:
I know, however <shell/shell.h> exists in zephyr as well so I had to workaround the file name collision. I guess full paths make such errors less likely to happen.

Reply:
> > this looks really good. thanks for all the work here.
> > would you mind terribly addressing the `lib` review comments?
> 
> @rwalker-apple not sure you've seen my comment, but I basically added "lib" component to the include directives to be able to distinguish <shell/shell.h> from CHIP from <shell/shell.h> from Zephyr. Let me know if that's ok or you think it's solveable in a different way.

I can't think of a quick fix that's better than yours.  We should think about whether chip's shell needs to be CHIPShell.h.  @turon any ideas?